lib/pages/profile/profile_header.dart

```diff
@@ -0,0 +1,127 @@
+                  radius: 36,
+                ),
+                 Column(children: [
+                  Text(SourceString.numPost,
+                    style: _textStyleBold(),),
+                  Text(SourceString.post,
+                    style: _textStyleNormal(),),
+                ],
+                ),
+
+                 Column(children: [
+                  Text(SourceString.numFollowers,
+                    style: _textStyleBold(),),
+                  Text(SourceString.followers,
+                    style: _textStyleNormal(),),
+                ],
+                ),
+
+                Column(children: [
+                  Text(SourceString.numFollowing,
+                    style: _textStyleBold(),),
+                  Text(SourceString.following,
+                    style: _textStyleNormal(),),
+                ],
+                ),
+              ]
+            ),
+            const SizedBox(height: 10),
+            Text(SourceString.userName,
+              style: _textStyleBold(),),
+            const Text(SourceString.yearOfBirth,
+              style: TextStyle(
+                  fontSize: 15,
+              ),),
+            const SizedBox(height: 10,),
+            _actionButtonProfile(),
+            const SizedBox(height: 10,),
+            const HighlightStories()
+          ],
+        ),
+      ),
+    );
+  }
+
+  Row _actionButtonProfile(){
+    return  Row(
+      children: [
+        Expanded(
+            child: TextButton(onPressed: () {  },
+              style: TextButton.styleFrom(
+                  backgroundColor: const Color(0x1F000000),
+                  foregroundColor: Colors.black,
+                  shape: RoundedRectangleBorder(
+                      borderRadius: BorderRadius.circular(12)
+                  )
+              ),
+              child: const Text(SourceString.editProfile),
+            )
+        ),
+        const SizedBox(width: 3,),
+        Expanded(
+            child: TextButton(onPressed: () {  },
+              style: TextButton.styleFrom(
+                  backgroundColor: const Color(0x1F000000),
+                  foregroundColor: Colors.black,
+                  shape: RoundedRectangleBorder(
+                      borderRadius: BorderRadius.circular(12)
+                  )
+              ),
+              child: const Text(
+                SourceString.shareProfile,
+                overflow: TextOverflow.ellipsis,
+              ),
+            )
+        ),
+        IconButton(
+            onPressed: () {},
+            icon: const FaIcon(FontAwesomeIcons.userPlus, size: 20,),
+            style: IconButton.styleFrom(
+              backgroundColor: const Color(0x1F000000),
+              shape: RoundedRectangleBorder(
+                borderRadius: BorderRadius.circular(12)
+              )
+            ),
+        )
+      ],
+    );
+  }
+  TextStyle _textStyleBold(){
```

Comment:
TextStyle should be applied to the whole app, you should mo to some where like AppTextStyle for everyone can reuse them

Reply:
i changed